package.json

```diff
@@ -16,6 +16,7 @@
     "jwt-encode": "^1.0.1",
     "miragejs": "^0.1.47",
     "react": "^18.2.0",
+    "react-css-modules": "^4.7.11",
```

Comment:
Why do we need this?

Reply:
When I initially did it without installing this, it gave me a lot of error, I then read that we need to install these dependencies for css modules to work in react.